### ISSUE TITLE
Timeouts of libmbus are too short on some cases

### ISSUE THREAD

**Issue by tsr8** [Apollon77/node-mbus#70]
In my project I need longer timeouts in libmbus to work properly. 

I can change source of libmbus and compile it and test it but:
how to compile node-mbus (need it in node-red)?

**Comment by Apollon77** [comment]
WHy you do not make a PR in the libmbus project to have it adjusted there?

**Comment by tsr8** [comment]
Thank you for the answer.
They already have special "serial-timeout" branch for this, which adds "-t" parameter. I have created an libmbus issue: https://github.com/rscada/libmbus/issues/186
I think that node-mbus shoud provide some setting to change timeouts easy way.

**Comment by Apollon77** [comment]
In fact thet PR is aloready included in their project and also in mine

https://github.com/Apollon77/libmbus/blob/build-windows/mbus/mbus-serial.c#L155

you need even higher values?

**Comment by tsr8** [comment]
> you need even higher values?

By testing, I need minimum 1100ms at 2400 baudrate. It fails with 1000ms.

**Comment by Apollon77** [comment]
yes so it only makes sense to make it configurable completely ... puhhh ...
